machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -24,6 +24,12 @@ If you do not have access to the {op-system-first} images that were used to crea
 
 Before you add more compute machines to a cluster that you installed on bare metal infrastructure, you must create {op-system} machines for it to use. You can either use an ISO image or network PXE booting to create the machines.
 
+[NOTE]
+====
+You must use the same ISO image that you used to install a cluster to deploy all new nodes in a cluster, before or after the cluster upgrades. It is recommended to use the same Ignition config file.
```

Comment:
The first time I read this sentence, I wasn't very clear to me what the "before or after the cluster upgrades" part meant. I wasn't sure if it was trying to convey that you have to use the same ISO image that you used on install, even after the cluster has been upgraded?

I'm just wondering if that part could be removed, since you do say at the end of the note "You can add the nodes before or after the updates."

Or if you do want to keep that part in the first sentence, would something like this mean the right thing: "... all new nodes in a cluster, even after the cluster has been upgraded." or something similar?

Reply:
yes I would want to keep the first part and re-phrase it.

Reply:
Actually on further insight, I have removed that bit `before or after the cluster upgrades`